src/components/UploadComponent/MultipleFileUploadField.tsx

```diff
@@ -0,0 +1,104 @@
+import { Grid, makeStyles } from '@material-ui/core';
+import { useField } from 'formik';
+import React, { useCallback, useEffect, useState } from 'react';
+import { FileError, FileRejection, useDropzone } from 'react-dropzone';
+import { SingleFileUploadWithProgress } from './SingleFileUploadWithProgress';
+import { UploadError } from './UploadError';
+
+let currentId = 0;
+
+function getNewId() {
+  // we could use a fancier solution instead of a sequential ID :)
+  return ++currentId;
+}
+
+export interface UploadableFile {
+  // id was added after the video being released to fix a bug
+  // Video with the bug -> https://youtube-2021-feb-multiple-file-upload-formik-bmvantunes.vercel.app/bug-report-SMC-Alpha-thank-you.mov
+  // Thank you for the bug report SMC Alpha - https://www.youtube.com/channel/UC9C4AlREWdLoKbiLNiZ7XEA
+  id: number;
+  file: File;
+  errors: FileError[];
+  url?: string;
+}
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
We don't write like that. We use styled-components for it. And we don't have theme! Rewrite it as in example below.
https://github.com/Intelligent-Cloud-System/ics-frontend/blob/dev/src/pages/Login/Login.styles.ts